src/BenchmarkDotNet/Toolchains/IDotNetFrameworkSdkProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BenchmarkDotNet.Toolchains
+{
+    public interface IDotNetFrameworkSdkProvider
```

Comment:
Creating another interface for this seems pointless. I don't think the interface is necessary at all (see comments in `DotNetSdkVersionValidator`).